core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -1512,6 +1520,7 @@ Map<String, OptimizelyDecision> decideAll(@Nonnull OptimizelyUserContext user,
     /**
      * Returns a decision result ({@link OptimizelyDecision}) for a given flag key and a user context,
      * skipping CMAB logic and using only traditional A/B testing.
+     * This will be called by mobile apps which will use non-blocking legacy ab-tests only (for backward compatibility with android-sdk)
```

Comment:
```suggestion
     * This will be called by mobile apps which will make synchronous decisions only (for backward compatibility with android-sdk)
```